### PR TITLE
Issue249

### PR DESCRIPTION
**Pull request by untzag**

### CODE REVIEW

File: WrightTools/data/_colors.py [hunk 1]
```diff
@@ -201,12 +201,12 @@ def from_COLORS(
...
-        channels['ai2'] = Channel(None, 'V', file_idx=18, name='ai2', label_seed=['2'])
-        channels['ai3'] = Channel(None, 'V', file_idx=19, name='ai3', label_seed=['3'])
-        channels['ai4'] = Channel(None, 'V', file_idx=20, name='ai4', label_seed=['4'])
-        channels['mc'] = Channel(None, None, file_idx=21, name='array', label_seed=['a'])
+        channels['ai0'] = Channel(None, file_idx=16, name='ai0', label_seed=['0'])
+        channels['ai1'] = Channel(None, file_idx=17, name='ai1', label_seed=['1'])
+        channels['ai2'] = Channel(None, file_idx=18, name='ai2', label_seed=['2'])
+        channels['ai3'] = Channel(None, file_idx=19, name='ai3', label_seed=['3'])
+        channels['ai4'] = Channel(None, file_idx=20, name='ai4', label_seed=['4'])
+        channels['mc'] = Channel(None, file_idx=21, name='array', label_seed=['a'])
```
Comment (ksunden) [inline WrightTools/data/_colors.py]: Shall we move name to be a positional argument, now that it is required?

File: WrightTools/data/_data.py [hunk 2]
```diff
@@ -215,14 +212,16 @@ def min(self):
 class Channel:
     """Channel."""
 
-    def __init__(self, values, units=None, file_idx=None, null=None, signed=None, name='channel',
-                 label=None, label_seed=None):
+    def __init__(self, values, name, units=None, file_idx=None, null=None, signed=None,
```
Comment (ksunden) [inline WrightTools/data/_data.py]: file_idx is intended to be removed in a future commit, correct?

File: WrightTools/data/_colors.py (hunk 1 above)
Comment (untzag) [reply WrightTools/data/_colors.py]: I'm too lazy---these lines of code are not long for this world anyway

File: WrightTools/data/_data.py (hunk 2 above)
Comment (untzag) [reply WrightTools/data/_data.py]: thanks

**Pull request merged**